index.php

```diff
@@ -1,8 +1,17 @@
 <?php
 
-require_once ('functions.php');
+require_once 'functions.php';
+require_once 'dbConnect.php';
 
+$db = connectdb();
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
```

Comment:
Charlie recommend placing this within your connectdb function in the dbConnect.php file